__tests__/reporter.js

```diff
@@ -0,0 +1,18 @@
+const {DefaultReporter} = require('@jest/reporters')
+
+/**
+ * A custom reporter that is silent (no console logs except for summary) when all tests have passed.
+ * Since our Image component may print desired warnings to console (used to inform the end user of best practice),
+ * we don't want them to clog the output.
+ * As jest prints not only the warning, but also where it originated, which for our build test means a very long
+ * output of gibberish that also takes a lot of compute time.
+ */
+class Reporter extends DefaultReporter {
```

Comment:
won't this confuse people who try to console log and see output during the tests?

Reply:
I've removed this reporter and instead mocked console.warn in the relevant responsive tests